recipes/wagyu/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagyu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0f0e4b78913e64083ca865f07e6aee30a24988890fc383f3003456287ed45d7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pybind11
+    - {{ compiler('cxx') }}
+    - conda-verify
```

Comment:
is `conda-verify` needed here? I don't see it being used upstream to build the package, although maybe I've missed something. I also don't think you need `pybind11` under `build`.

Reply:
It was there as conda build was warning it was missing. I have removed as it's not needed for this recipe specifically.